pixel-saver@deadalnix.me/buttons.js

```diff
@@ -229,9 +232,28 @@ function init(extensionMeta) {
 	extensionPath = extensionMeta.path;
 }
 
+function loadOptions() {
+	if (options == null) {
+		options = new Settings.Options();
+		options.SHOW_BUTTONS.changed(function() {
+			if (options.SHOW_BUTTONS.get()) {
+				showButtons();
+			} else {
+				hideButtons();
+			}
+		});
+	}
```

Comment:
Initially I put the callback into settings.js as well (see https://github.com/deadalnix/pixel-saver/pull/56/commits/c6f808c1ad2a956725a114bfd8c295aac8670df8), but then I'd thought why not handle everything in the buttons module? This way all button related logic is in just one place.

But I have no strong opinion either way. I can revert to the old approach. No problem.